doc/api/esm.md

```diff
@@ -501,6 +501,19 @@ a nested conditional does not have any mapping it will continue checking
 the remaining conditions of the parent condition. In this way nested
 conditions behave analogously to nested JavaScript `if` statements.
 
+#### Resolving custom conditions
+
+When running Node.js, custom comma-separated conditions can be set with the
```

Comment:
"can be set" implies to me that it replaces the defaults; is it correct that the semantics here are that these are conditions to *prepend* to the standard list, which defaults to an empty list?

Reply:
Conditions themselves have no ordering. But we could use a term like _added_ rather than _set_.